app/scripts/formentry/formschema/form1.json

```diff
@@ -2566,8 +2576,8 @@
                   "validators": [
                     {
                       "type": "js_expression",
-                      "failsWhenExpression": "!(isEmpty(qPcptoxicity)&&isEmpty(myValue))&&(qPcptoxicity==='a8aaf3e2-1350-11df-a1f1-0026b9348838')",
-                      "message": "You indicated patient is changing /changing dose/substituting/stopping arvs due to other toxicity, indicate the other cause."
+                      "failsWhenExpression": "!isEmpty(qPcptoxicity)&&isEmpty(myValue)&&qPcptoxicity==='a8aaf3e2-1350-11df-a1f1-0026b9348838'",
```

Comment:
Spaces between logical operators and expression would be awesome. For example instead of a&&b it should be
a && b